src/js/components/FormField/formfield.stories.js

```diff
@@ -133,10 +134,42 @@ const FormFieldHelpError = props => (
   </Grommet>
 );
 
+const customFormFieldTheme = {
+  global: {
+    font: {
+      size: '13px',
+    },
+    input: {
+      weight: 400,
+    },
+  },
+  formField: {
+    label: {
+      color: 'dark-3',
+      size: 'xsmall',
+      margin: { vertical: 0, bottom: 'small', horizontal: 0 },
+      weight: '600',
+    },
+    border: false,
+    margin: 0,
+  },
+};
+
+const CustomFormField = () => (
+  <Grommet theme={deepMerge(grommet, customFormFieldTheme)}>
+    <Box align="center" pad="large">
+      <FormField label="Label" htmlFor="text-area">
+        <TextArea id="text-area" placeholder="placeholder" />
+      </FormField>
+    </Box>
+  </Grommet>
+);
+
 storiesOf('FormField', module)
   .add('TextInput', () => <FormFieldTextInput />)
   .add('TextArea', () => <FormFieldTextArea />)
   .add('Select', () => <FormFieldSelect />)
   .add('CheckBox', () => <FormFieldCheckBox />)
   .add('Toggle', () => <FormFieldToggle />)
-  .add('Help and error', () => <FormFieldHelpError />);
+  .add('Help and error', () => <FormFieldHelpError />)
+  .add('Custom Theme', () => <CustomFormField />);
```

Comment:
Thanks for adding a story!
I noticed these warnings when running it:
![Uploading Screen Shot 2019-02-13 at 2.21.20 PM.png…]()

Reply:
let me take a look...